tests/package_managers/.gitignore

```diff
@@ -0,0 +1,5 @@
+BUILD.bazel
```

Comment:
These are unusual entries in a git ignore, so could you comment explaining why?

Reply:
Done.